sql/core/src/main/scala/org/apache/spark/sql/execution/CommandResultExec.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.catalyst.expressions.{Attribute, UnsafeProjection}
+import org.apache.spark.sql.catalyst.plans.QueryPlan
+import org.apache.spark.sql.execution.metric.SQLMetrics
+
+/**
+ * Physical plan node for holding data from a command.
+ *
+ * `rows` may not be serializable and ideally we should not send `rows` to the executors.
+ * Thus marking it as transient.
+ */
+case class CommandResultExec(
```

Comment:
with InputRDDCodegen？

Reply:
We just execute command in driver locally and no RDD here.

Reply:
We should match `LocalTableScanExec` and implement `InputRDDCodegen`, to support whole stage codegen.

Reply:
Is it similar to LocalTableScanExec?

Reply:
OK